doc/whats_new/v1.2.rst

```diff
@@ -42,7 +42,13 @@ random sampling procedures.
   :func:`linear_model._sgd_fast._plain_sgd` which is used by :class:`linear_model.SGDRegressor`
   and :class:`linear_model.SGDClassifier`. The old condition did not disambiguate between
   training and validation set and had an effect of overscaling the error tolerance.
-  This has been fixed in :pr:`23798` by :user:`Harsh Agrawal <Harsh14901>`
+  This has been fixed in :pr:`23798` by :user:`Harsh Agrawal <Harsh14901>`.
+
+- |Fix| :class:`model_selection.GridSearchCV` and
+  :class:`model_selection.RandomizedSearchCV` now rank fit/scoring routines
+  that raise an exception as equal worst. Previously they were ranked as worst performers
+  in an arbitrary order.
+  :pr:`24543` by :user:`Guillaume Lemaitre <glemaitre>`.
```

Comment:
My attempt (I think it is more about nan scores than fit/scoring routines that raise an exception). Feel free to ignore or adapt:
```suggestion
- |Fix| For :class:`model_selection.GridSearchCV` and
  :class:`model_selection.RandomizedSearchCV` ranks corresponding to nan
  scores will all be set to the maximum possible rank.
  :pr:`24543` by :user:`Guillaume Lemaitre <glemaitre>`.
```